app/main.py

```diff
@@ -1,2 +1,121 @@
+from typing import Any
+
+
+class HashTable:
```

Comment:
This is not really a hash table - it is a node that is inside of  a hash table which is `self.table` inside your dicitionary. Also it says in task requirements 

> Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.

Also this is basically a part of a Dictionary class and will not be used outside of it so it would make sense to declare it inside of class dictionary